### PR TITLE
VZ-10223: Update pod startup command to check if plugin is already installed

### PR DESCRIPTION
**Pull request by saketmht**

### CODE REVIEW

File: pkg/resources/helper.go [hunk 1]
```diff
@@ -63,29 +63,62 @@ const (
 	echo "Commenting out java heap settings in jvm.options..."
 	sed -i -e '/^-Xms/s/^/#/g' -e '/^-Xmx/s/^/#/g' config/jvm.options
 	`
+	PluginInstalledCheckWithExit = `
+     exit_status=$?
+     if [ $exit_status -ne 0 ]; then
+          if grep -iq "plugin .* already exists" /tmp/error.log; then
+               echo "Plugin already installed. Skipping installation."
```
Comment (adiforluls) [inline pkg/resources/helper.go:70]: Should you be cleaning temp directory here?

File: pkg/resources/helper.go [hunk 2]
```diff
@@ -63,29 +63,62 @@ const (
...
+               cat /tmp/error.log
+               exit 1
+          fi
+     fi
+    `
+	PluginInstalledCheck = `
+     exit_status=$?
+     if [ $exit_status -ne 0 ]; then
+          if grep -iq "plugin .* already exists" /tmp/error.log; then
+               echo "Plugin already installed. Skipping installation."
```
Comment (adiforluls) [inline pkg/resources/helper.go:81]: Same question about temp directory clean up.

File: pkg/resources/helper.go [hunk 3]
```diff
@@ -63,29 +63,62 @@ const (
...
+          else
+               cat /tmp/error.log
+          fi
+     fi
+    `
+	DashboardsPluginInstalledCheck = `
+     exit_status=$?
+     if [ $exit_status -ne 0 ]; then
+          if grep -iq "plugin .* already exists" /tmp/error.log; then
+               echo "Plugin already installed. Skipping installation"
```
Comment (adiforluls) [inline pkg/resources/helper.go:91]: Same question about temp directory cleanup.

File: pkg/resources/helper.go (hunk 1 above)
Comment (saketmht) [reply pkg/resources/helper.go:70]: Added a line below to clean up the temp file being created

File: pkg/resources/helper.go (hunk 2 above)
Comment (saketmht) [reply pkg/resources/helper.go:82]: Updated

File: pkg/resources/helper.go (hunk 3 above)
Comment (saketmht) [reply pkg/resources/helper.go:93]: Updated

**Review by adiforluls — APPROVED** [review]

**Pull request merged**